crates/starknet/src/starknet/starknet_config.rs

```diff
@@ -57,6 +66,7 @@ impl Default for StarknetConfig {
             dump_on: None,
             dump_path: None,
             re_execute_on_init: true,
+            state_archive: StateArchiveCapacity::None,
```

Comment:
Can you instead do `StateArchiveCapacity::default()`? If we ever change it, it would be good that it's changed in just one place. I think if you derive Default, it will use the first enum option as the default (unless specified otherwise) - which is what we need (`none`).

Reply:
Fixed in c727b6cd25e1aaa1debbc6ded48ac22872545c48